src/api/Tasks.js

```diff
@@ -195,6 +196,27 @@ class Tasks extends Base {
             errorCallback,
         });
     }
+
+    /**
+     * Overrides the Base.successHandler to handle filling in a placeholder user if
+     * task created_by or tas assignment assigned_to field is null
+     *
+     * @param {Object} data the api response data
+     */
+    successHandler = (data: Object): void => {
+        if (this.isDestroyed() || typeof this.successCallback !== 'function') {
+            return;
+        }
+
+        if (!data.entries) {
+            this.successCallback(data);
```

Comment:
If there isnt an entries array, should be be an error? Or should you return some default structure, since data is incorrect

Reply:
I'll change it to return an empty default object `{ entries: [], total_count: 0 }`